rascal-lsp/src/main/rascal/lang/rascal/lsp/refactor/WorkspaceInfo.rsc

```diff
@@ -66,7 +69,8 @@ data Cursor
 
 alias MayOverloadFun = bool(set[loc] defs, map[loc, Define] defines);
 alias FileRenamesF = rel[loc old, loc new](str newName);
-alias DefsUsesRenames = tuple[set[loc] defs, set[loc] uses, FileRenamesF renames];
+data RenameLocation = rl(loc l, Maybe[ChangeAnnotationId] annotation = nothing());
```

Comment:
smells like a tuple. is there a reason why it's an adt? 

```
tuple[rel[loc, Maybe[ChangeAnnotationId]] defs...
```

would have been possible?

it reads to me as a bit messy to annotate the locations like this.

Reply:
You're right. It's just a tuple. Refactored!